web/src/stream_color.js

```diff
@@ -7,8 +8,23 @@ import * as message_view_header from "./message_view_header";
 import * as settings_data from "./settings_data";
 import * as stream_settings_ui from "./stream_settings_ui";
 
+extend([lchPlugin]);
+
+export function get_stream_privacy_icon_color(color) {
```

Comment:
Same request for a comment explaining what the algorithm is supposed to do, since it's a bit opaque.

Reply:
Okay, added.